src/material/core/style/_focus-indicators.scss

```diff
@@ -0,0 +1,68 @@
+/// Mixin that turns on strong focus indicators.
+///
+/// @example
+///   .my-app {
+///     @include mat-strong-focus-indicators($config);
+///   }
```

Comment:
Sorry to make you shift comments around! But this comment should be above the `strong-focus-indicators` mixin, not this one! I think we should also change the comment from `@include mat-strong-focus-indicators($config);` to `@include mat.strong-focus-indicators($config)`.

Reply:
Ah gotcha! Noticed your previous comment about removing this comment from `src/material/core/focus-indicators.scss`, so can just remove this comment entirely?